ocaml/xapi/xapi_ha_vm_failover.ml

```diff
@@ -175,20 +187,407 @@ let order_f (_, vm_rec) =
 
 let ( $ ) x y = x y
 
-(*****************************************************************************************************)
-(* Planning code follows                                                                             *)
-
 (* Compute the total memory required of a VM (Running or not) *)
 let total_memory_of_vm ~__context policy snapshot =
   let main, shadow =
     Memory_check.vm_compute_start_memory ~__context ~policy snapshot
   in
   Int64.add main shadow
 
+(* Used to sort pairs into increasing order of their second component *)
+let bigger_than (_, a) (_, b) = compare a b
```

Comment:
Could it be named `compare` as well?

Reply:
The `bigger_than` can't be renamed to `compare`, as compare it also used here: https://github.com/gangj/xen-api/blob/private/gangj/CP-48625/ocaml/xapi/xapi_ha_vm_failover.ml#L351